slave-base/contrib/bin/run-jnlp-client

```diff
@@ -4,6 +4,14 @@
 # will use for the auto-discovery of this slave.
 #
 
+# The directory that Jenkins will execute the builds and store cache files.
+# The directory has to be writeable for the user that the container is running
+# under.
+export JENKINS_HOME=/home/jenkins
+
+# Make sure the Java clients have valid $HOME directory set
+export HOME=${JENKINS_HOME}
```

Comment:
i don't understand why this should be necessary:
https://github.com/openshift/jenkins/blob/master/slave-base/Dockerfile#L5